grammar.js

```diff
@@ -92,9 +91,14 @@ module.exports = grammar({
 
     // A function, generator, class, or variable declaration
     _declaration: $ => prec(PREC.DECLARATION, choice(
-      $.function,
-      $.generator_function,
-      $.class,
+      seq(
```

Comment:
Yeah, I added this was in order to make functions and classes parse as declarations rather than expression statements. When parsing this code:

```js
1| function foo() {}
2| var x
```

the lexer produces an `_automatic_semicolon` token at the end of line 1. Without this change, that token would cause *only* the `expression_statement` interpretation to be valid, not the `_declaration`.